lib/modules/versioning/unity3d/index.ts

```diff
@@ -0,0 +1,81 @@
+import semver from 'semver';
+import { regEx } from '../../../util/regex';
+import { GenericVersion, GenericVersioningApi } from '../generic';
+import type { VersioningApi } from '../types';
+
+export const id = 'unity3d';
+export const displayName = 'Unity3D';
+export const urls = [
+  'https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html#version-define-expressions:~:text=characters%20are%20supported.-,Unity%20version%20numbers,-Current%20versions%20of',
```

Comment:
```suggestion
  'https://docs.unity3d.com/Manual/ScriptCompilationAssemblyDefinitionFiles.html#version-define-expressions',
```
don't add chrome specific highlighting